acme/acme/messages.py

```diff
@@ -251,7 +252,7 @@ class Registration(ResourceBody):
     contact = jose.Field('contact', omitempty=True, default=())
     agreement = jose.Field('agreement', omitempty=True)
     status = jose.Field('status', omitempty=True)
-    terms_of_service_agreed = jose.Field('terms-of-service-agreed', omitempty=True)
+    terms_of_service_agreed = jose.Field('termsOfServiceAgreed', omitempty=True)
```

Comment:
same here -- do we not need the v1 version?

Reply:
Neither `terms-of-service-agreed` nor `termsOfServiceAgreed` is in v1 (it uses `agreement` instead).

Reply:
ah I see now, this is updating something that was added in #5287.